drivers/display/ssd1306_regs.h

```diff
@@ -28,8 +28,8 @@
 #define SSD1306_SET_ENTIRE_DISPLAY_OFF		0xa4
 #define SSD1306_SET_ENTIRE_DISPLAY_ON		0xa5
 
-#define SSD1306_SET_NORMAL_DISPLAY		0xa6
-#define SSD1306_SET_REVERSE_DISPLAY		0xa7
+#define SSD1306_SET_NORMAL_DISPLAY		0xa6 /**< RAM data of 1 indicates an "ON" pixel */
+#define SSD1306_SET_REVERSE_DISPLAY		0xa7 /**< RAM data of 0 indicates an "ON" pixel */
```

Comment:
Excuse me, but why did you come up with the idea of putting those not-so-nice comments on the side of the definition instead of above it, and using doxygen style for a header file that does not define any API and is only used by the driver, it is the driver's private header, and there is absolutely no reason to expose it to any kind of documentation?

Reply:
removed doxygen style and moved it to line above